src/main/msp/msp.c

```diff
@@ -1186,9 +1187,11 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, getMotorCount());
         sbufWriteU8(dst, motorConfig()->motorPoleCount);
 #ifdef USE_DSHOT_TELEMETRY
+        sbufWriteU8(dst, motorConfig()->dev.useDshotTelemetry);
         sbufWriteU8(dst, useDshotTelemetry);  // DSHOT telemetry is enabled
```

Comment:
We are not adding the same parameter twice? The first was added by @etracer65 when we implement the RPM into the motors tab, using the Dshot Telemetry.

Reply:
I only see one. The other is only sends invalidPct via msp.

Reply:
Yes we are indeed - I had assumed that `useDshotTelemetry` was feedback on bidirectional Dshot actually being active. Well spotted! :eyes:

Reply:
Oh right there. Yes!